test/trino.test.ts

```diff
@@ -51,6 +51,56 @@ describe('TrinoFormatter', () => {
     `);
   });
 
+  it('formats row PATTERN()s', () => {
+    const result = format(`
+      SELECT * FROM orders MATCH_RECOGNIZE(
+        PARTITION BY custkey
+        ORDER BY orderdate
+        MEASURES
+                  A.totalprice AS starting_price,
+                  LAST(B.totalprice) AS bottom_price,
+                  LAST(U.totalprice) AS top_price
+        ONE ROW PER MATCH
+        AFTER MATCH SKIP PAST LAST ROW
+        PATTERN ((A | B){5} {- C+ D+ -} E+)
+        SUBSET U = (C, D)
+        DEFINE
+                  B AS totalprice < PREV(totalprice),
+                  C AS totalprice > PREV(totalprice) AND totalprice <= A.totalprice,
+                  D AS totalprice > PREV(totalprice)
+        )
+    `);
+    expect(result).toBe(dedent`
+      SELECT
+        *
+      FROM
+        orders
+      MATCH_RECOGNIZE
+        (
+          PARTITION BY
+            custkey
+          ORDER BY
+            orderdate
+          MEASURES
+            A.totalprice AS starting_price,
+            LAST (B.totalprice) AS bottom_price,
+            LAST (U.totalprice) AS top_price
```

Comment:
Should actually be

```
            LAST(B.totalprice) AS bottom_price,
```

Seems like these are documented at https://trino.io/docs/current/sql/match-recognize.html#row-pattern-recognition-expressions

Will adding these as both functions and keywords mess up formatting of the keywords? I don't people are really actually using this row pattern stuff, so we should prefer `LAST` as a keyword than a function if that's the case.

Reply:
Well, the main question is, how else is the `LAST` keyword used in Trino? I found three usages:

- `SORT BY col NULLS LAST`
- `AFTER MATCH SKIP TO LAST ROW`
- `AFTER MATCH SKIP TO LAST identifier`

Importantly there doesn't seem to be any cases where we'd want `LAST` followed by `(` to be formatted as `LAST (`. Therefore it should be safe to list it under functions.